index.bs

```diff
@@ -5636,13 +5643,17 @@ To <dfn>store a record into an object store</dfn> with
 To <dfn>retrieve a value from an object store</dfn> with
 |targetRealm|, |store| and |range|, run these steps:
 
+NOTE:
+    May return a {{DOMException}} if an error occurs while running these steps.
```

Comment:
Avoid "may" as that has [RFC 2119](https://datatracker.ietf.org/doc/html/rfc2119) meaning, i.e. it means that the specified behavior is optional for implementations. Also, it should be normative.

https://infra.spec.whatwg.org/#algorithm-declaration has the preferred formulation, which is:

> ..., perform the following steps. They return ...

So maybe:

> ... run these steps. They return a value from the object store, or an error (a DOMException).

Reply:
Thank you for these pointers! I've updated the algorithm prose accordingly. https://infra.spec.whatwg.org/#algorithm-declaration specifies mentioning the _return type_ and not a description of the returned value, so I have used `ECMAScript value` instead and also included `undefined` in the list of possible return types. Please LMK if I've interpreted any of this incorrectly.